tgui/packages/tgui/interfaces/Atm.js

```diff
@@ -0,0 +1,30 @@
+import { useBackend } from "../backend";
+import { Box, Button, Section, TextInput, NoticeBox, NumberInput } from "../components";
+import { Window } from "../layouts";
+
+export const ATM = (props, context) => {
```

Comment:
```suggestion
export const Atm = (props, context) => {
```
The game is looking for `Atm`, you're exporting `ATM`